src/main/java/pene/gc/importer/utils/Datareader.java

```diff
@@ -23,7 +23,10 @@ public static int getAvatarId(String name){
         catch (IOException e) {
             e.printStackTrace();
         }
-
+        catch (NullPointerException e) {
+            Grasscutter.getLogger().warn("Possibly unknown avatar: " + name);
+            e.printStackTrace();
+        }
         return 0;
```

Comment:
It would be also good to add some if statements when returned id for avatars/weapons/etc == 0

Reply:
Will take this into account, if the need for updating plugin arises.